### PR TITLE
Components

### PR DESCRIPTION
**Pull request by lorenzoworx**
In this pull request, I have developed a React component that will hold the core functionality of the math-magicians webapp; a calculator.

### CODE REVIEW

File: src/components/calculator.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  justify-content: center;
+  align-items: flex-end;
+  background-color: grey;
+  color: white;
+  padding: 10px;
+}
+
+.previous {
+  font-size: 24px;
+  margin-bottom: 10px;
```
Comment (t-yanick) [inline src/components/calculator.css]: - [x] Kindly remove all unused and commented pieces of code from your codebase. This makes your codebase clean and professional.

Comment (t-yanick) [inline src/components/calculator.css]: - [x] Kindly remove all unused and commented pieces of code from your codebase. This makes your codebase clean and professional.

File: src/components/calculator.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  display: grid;
+  grid-template-rows: auto auto repeat(5, 1fr);
+  grid-template-columns: repeat(4, 1fr);
+  width: 500px;
+  height: 400px;
+  grid-gap: 5px;
+  padding: 10px;
+  border: 1px solid black;
+}
+
```
Comment (t-yanick) [inline src/components/calculator.css:9]: - [x] As per [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) you are not supposed to add any extra design. But when I compare your design with the given design, I see differences in
- the gaps between the buttons and the border
- the orange color you have used for the operation buttons
- the display of the answer has two zeroes, one up and one down
Check the attached screenshots below and make your design to match the given design
- Given design
image

- Your design
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @lorenzoworx , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Components directory present with calculator.js file inside.
:heavy_check_mark: Calculator component is functional.
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] Great job working on this milestone. When I click on your [MIT](https://github.com/lorenzoworx/math-magicians/blob/components/LICENSE) license link, it redirects me to an error 404 page. Kindly fix this issue as it is not professional to keep broken links in our README.
- [x] As per the project requirements, you are supposed to remove all boilerplate created by CRA, but when I check your project folders, I still find logo.svg, reportWebVitals.js and setupTests.js. Kindly remove them.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.css (hunk 2 above)
Comment (lorenzoworx) [reply src/components/calculator.css:9]: @t-yanick Hey can you help me out with the color code for the orange used? I have no idea how to determine what specific color was used

**Comment by t-yanick** [comment]
- [x] You can use this color [picker](https://imagecolorpicker.com/en) and do a screenshot of a button and upload on the site to get the exact color. Hope this helps.

File: src/components/calculator.css (hunk 2 above)
Comment (t-yanick) [reply src/components/calculator.css:9]: - [x] You can use this color [picker](https://imagecolorpicker.com/en) and do a screenshot of a button and upload on the site to get the exact color. Hope this helps.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @lorenzoworx, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**